### PR TITLE
add in post about tooling summit

### PR DESCRIPTION
**Pull request by ckipp01**

### CODE REVIEW

File: blog/_posts/2023-04-06-march-2023-scala-tooling-summit.md [hunk 1]
```diff
@@ -0,0 +1,357 @@
+---
+layout: blog-detail
+post-type: blog
+by: Chris Kipp and the Scala Center team
+title: March 2023 - Scala Tooling Summit
+description: A recap of the recent Scala Tooling Summit in Lausanne, Switzerland.
+---
+
+<img style="max-height: 350px; display: block; margin: auto;" src="{{ site.baseurl }}/resources/img/march-2023-tooling-summit-stairs.jpg" alt="Scala stairs with people standing on the stairs">
```
Comment (ckipp01) [inline blog/_posts/2023-04-06-march-2023-scala-tooling-summit.md:9]: I don't love this, but the styling for images in blogs is massive. I saw in others that we seem to rely on inline styling, so I just followed suit here. However, we should probably change this up in the future and actually correctly style images with css.

This ends up looking like this:
Screenshot 2023-04-06 at 16 08 44

File: blog/_posts/2023-04-06-march-2023-scala-tooling-summit.md [hunk 2]
```diff
@@ -0,0 +1,357 @@
...
+- We’d like to ensure there is solid single-file support in various editors when
+  using scala-cli. This is being looked into both in IntelliJ and Metals
+
+### A discussion around BSP
+
+The [Build Server Protocol (BSP)](https://build-server-protocol.github.io/) was
+envisioned as a unified way to compile, run, test, etc your code in a way that
+doesn’t make build clients or servers re-implement integrations for every tool
+similar to how the Language Server Protocol
+(LSP)[https://microsoft.github.io/language-server-protocol/) works with language
```
Comment (jastice) [inline blog/_posts/2023-04-06-march-2023-scala-tooling-summit.md:210]: link brackets are wrong way around

File: blog/_posts/2023-04-06-march-2023-scala-tooling-summit.md [hunk 3]
```diff
@@ -0,0 +1,357 @@
...
+which aren’t easily migrated. The main outcome of this discussion was simply
+better understanding the challenges involved, so that if someone is willing to
+put in the time here, we have a starting point and some research done.
+
+### IDE support in “polyglot” projects that contain Scala
+
+Traditionally IntelliJ has been the leader in this front offering great support
+for mixed JVM-language projects. However this is made trickier with tools that
+may not always be well-supported in this context like sbt or Mill. With the
+upcoming new [BSP plugin](https://lp.jetbrains.com/new-bazel-plugin/) for
```
Comment (jastice) [inline blog/_posts/2023-04-06-march-2023-scala-tooling-summit.md:272]: I'd prefer not linking it publicly yet

Comment (ckipp01) [reply blog/_posts/2023-04-06-march-2023-scala-tooling-summit.md:272]: Alright, just to make sure can I still mention there is a new BSP plugin coming?

File: blog/_posts/2023-04-06-march-2023-scala-tooling-summit.md (hunk 2 above)
Comment (ckipp01) [reply blog/_posts/2023-04-06-march-2023-scala-tooling-summit.md:210]: Ah good catch.

File: blog/_posts/2023-04-06-march-2023-scala-tooling-summit.md [hunk 4]
```diff
@@ -0,0 +1,357 @@
...
+
+### The state of editor support for scala-cli
+
+Since [scala-cli](https://scala-cli.virtuslab.org/) is still quite new, much of
+the conversation during this topic was getting all parties involved on the same
+page about how scala-cli works, what limitations currently exist, and how we can
+collectively tackle them. Here are a few action items and links from the
+discussion:
+
+- [High-level IntelliJ Scala Plugin issue for supporting
```
[the remark this answers is not in the document]
Comment (ckipp01) [reply blog/_posts/2023-04-06-march-2023-scala-tooling-summit.md:197]: Out of curiosity, can I ask why? I do think people will be happy to know there is an issue tracking this that they can click and go to in order to add comments, etc. I can for sure take it off, but I know that scala-cli support in IntelliJ is a highly sought-after feature.

File: blog/_posts/2023-04-06-march-2023-scala-tooling-summit.md (hunk 3 above)
Comment (jastice) [reply blog/_posts/2023-04-06-march-2023-scala-tooling-summit.md]: yes, that's fine

File: blog/_posts/2023-04-06-march-2023-scala-tooling-summit.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (ckipp01) [reply blog/_posts/2023-04-06-march-2023-scala-tooling-summit.md:197]: Updated the wording here to what we spoke about on Slack.

File: blog/_posts/2023-04-06-march-2023-scala-tooling-summit.md [hunk 5]
```diff
@@ -0,0 +1,353 @@
...
+While no concrete action has been taken yet, the conversation has started and
+will be sure to continue moving forward.
+
+### Standardizing Scala worksheet implementations
+
+Currently in the Scala ecosystem we have 3 different approaches to worksheets:
+
+- IntelliJ worksheets (plain and interactive mode) which re-use functionality from the Scala REPL
+- Metals worksheets using [mdoc](https://scalameta.org/mdoc/) as an engine
+- Scastie which uses a custom Macro implementation
```
Comment (julienrf) [inline blog/_posts/2023-04-06-march-2023-scala-tooling-summit.md:170]:
```suggestion
- Scastie which uses a custom macro implementation
```

File: blog/_posts/2023-04-06-march-2023-scala-tooling-summit.md [hunk 6]
```diff
@@ -0,0 +1,353 @@
...
+participants and movement already visible on the topics above, we feel confident
+that these efforts will result in a more stable, feature-rich, and unified
+tooling ecosystem. The Scala Center will be continuing to facilitate and plan
+further collaboration related to these topics and efforts.
+
+We’re also looking ahead to another summit that will coincide with the upcoming
+[Scala Days](https://scaladays.org/) in Madrid. We’ll follow up on the
+discussion topics above and also dive into new ones.
+
+There will be a discussion surrounding the content of this post of the
```
Comment (julienrf) [inline blog/_posts/2023-04-06-march-2023-scala-tooling-summit.md:351]:
```suggestion
There will be a discussion surrounding the content of this post on the
```

?

**Review by julienrf — APPROVED** [review]
Thank you, this looks good to me!

**Comment by julienrf** [comment]
You will have to update the publication date to today in the file name.

File: blog/_posts/2023-04-06-march-2023-scala-tooling-summit.md (hunk 6 above)
Comment (ckipp01) [reply blog/_posts/2023-04-06-march-2023-scala-tooling-summit.md:351]: Ah yup, fixed.

**Comment by ckipp01** [comment]
> You will have to update the publication date to today in the file name.

Done.

@jastice or @unkarjedy, any final input from you both? Or an you give this a ✅ .

**Comment by SethTisue** [comment]
@ckipp01 in Safari, Chrome, and Firefox on my Mac the image renders all stretched

Screenshot 2023-04-11 at 3 01 42 PM

**Comment by julienrf** [comment]
Same for me (Linux). The problem can be solved by adding `width: auto` to the `style` property.

**Comment by lrytz** [comment]
... or was it the fondue that changed our proportions?

**Comment by ckipp01** [comment]
Haha, not just the fondue. Thanks for the heads up, all fixed in https://github.com/scala/scala-lang/pull/1485.

**Comment by unkarjedy** [comment]
good from me

**Pull request merged**